### ISSUE TITLE
Uncaught exception in `/api/photometry`

### ISSUE THREAD

**Issue by stefanv** [skyportal/skyportal#2667]
```
ERROR:tornado.application:Uncaught exception POST /api/photometry (131.215.198.70)
  File "/skyportal/skyportal/handlers/api/photometry.py", line 290, in standardize_photometry_data
    first_offender = np.argwhere(missing)[0, 0]
ValueError: Data must be 1-dimensional
```

**Comment by guynir42** [comment]
@stefanv I'm not sure how this happens. The bug comes from the lines right before: 
```
        for field in PhotMagFlexible.required_keys:
            missing = df[field].isna()
            if any(missing):
                first_offender = np.argwhere(missing)[0, 0]
```
Somehow `df[field].isna()` produces something that is not 1-dimensional. I don't think an empty array would get past the `if any(missing)` so it is likely that indexing by `field` somehow gets a multidimensional dataframe. I just don't have a good guess why that would happen... or what to do if it does (raise? serialize the result into a 1D array?)

**Comment by stefanv** [comment]
Here's the full traceback:

```
HTTPServerRequest(protocol='http', host='fritz.science', method='POST', uri='/api/photometry', version='HTTP/1.0', remote_ip='131.215.198.70')
Traceback (most recent call last):
  File "/skyportal_env/lib/python3.8/site-packages/tornado/web.py", line 1702, in _execute
    result = method(*self.path_args, **self.path_kwargs)
  File "/skyportal/baselayer/app/access.py", line 32, in wrapper
    return method(self, *args, **kwargs)
  File "/skyportal/baselayer/app/access.py", line 61, in wrapper
    return method(self, *args, **kwargs)
  File "/skyportal/skyportal/handlers/api/photometry.py", line 701, in post
    df, instrument_cache = self.standardize_photometry_data()
  File "/skyportal/skyportal/handlers/api/photometry.py", line 290, in standardize_photometry_data
    first_offender = np.argwhere(missing)[0, 0]
  File "<__array_function__ internals>", line 5, in argwhere
  File "/skyportal_env/lib/python3.8/site-packages/numpy/core/numeric.py", line 618, in argwhere
    return transpose(nonzero(a))
  File "<__array_function__ internals>", line 5, in nonzero
  File "/skyportal_env/lib/python3.8/site-packages/numpy/core/fromnumeric.py", line 1921, in nonzero
    return _wrapfunc(a, 'nonzero')
  File "/skyportal_env/lib/python3.8/site-packages/numpy/core/fromnumeric.py", line 54, in _wrapfunc
    return _wrapit(obj, method, *args, **kwds)
  File "/skyportal_env/lib/python3.8/site-packages/numpy/core/fromnumeric.py", line 47, in _wrapit
    result = wrap(result)
  File "/skyportal_env/lib/python3.8/site-packages/pandas/core/generic.py", line 2025, in __array_wrap__
    return self._constructor(res, **d).__finalize__(  # type: ignore[arg-type]
  File "/skyportal_env/lib/python3.8/site-packages/pandas/core/series.py", line 439, in __init__
    data = sanitize_array(data, index, dtype, copy)
  File "/skyportal_env/lib/python3.8/site-packages/pandas/core/construction.py", line 576, in sanitize_array
    subarr = _sanitize_ndim(subarr, data, dtype, index, allow_2d=allow_2d)
  File "/skyportal_env/lib/python3.8/site-packages/pandas/core/construction.py", line 627, in _sanitize_ndim
    raise ValueError("Data must be 1-dimensional")
ValueError: Data must be 1-dimensional

```

**Comment by stefanv** [comment]
I see something similar with this example:

```
df = pd.DataFrame.from_dict({'a': [1,2,np.nan], 'b': [4,5,6]})
missing = df['a'].isna()
np.argwhere(missing)
```

Is fixed by using `missing.values` instead in the `argwhere` call.

**Comment by guynir42** [comment]
I will try to do that, but since I'm not sure how to recreate the bug, we will just need to wait and see if that works.

**Comment by guynir42** [comment]
Closed by #2675 (I hope)
